spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/actions/BaseDeleteOrphanFilesSparkAction.java

```diff
@@ -32,6 +32,7 @@
 import java.util.function.Consumer;
 import java.util.function.Predicate;
 import java.util.stream.Collectors;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
We don't use anything from Apache commons directly in Iceberg. It may be available in the classpath, but we don't want a direct dependency on it.

Reply:
Using Guava Strings now